mycpp/gc_builtins.h

```diff
@@ -100,6 +100,13 @@ class OSError : public IOError_OSError {
   }
 };
 
+class SystemExit : public _ExceptionOpaque {
```

Comment:
I think this is probably unused given the comment in `core/completion.py`, but it's probably hard to `if mylib.PYTHON` out ...